readme.md

```diff
@@ -13,35 +13,64 @@ $ npm install p-progress
 ## Usage
 
 ```js
-const PProgress = require('p-progress');
+import {PProgress} from 'p-progress';
```

Comment:
The main example should be the `pProgress` default export. This example can be in the API docs for `new PProgress`.